app/grandchallenge/evaluation/templates/evaluation/evaluation_detail.html

```diff
@@ -309,6 +309,17 @@ <h3 class="card-title">Visibility</h3>
                             </button>
                         </form>
                     {% endif %}
+
+                    {% if object.missing_metrics %}
+                        <br>
+                        <div class="alert alert-warning mb-0">
+                            This evaluation has missing metric(s) from metrics.json {{ object.missing_metrics }}
```

Comment:
I think I would prefer something like `The metrics.json output file for this evaluation is missing the following metrics: ... `